### PR TITLE
chore(ngLocale): regenerate locale files

### PR DESCRIPTION
**Pull request by lgalfaso**

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -123,9 +123,6 @@ module.exports = function(grunt) {
       ngCookies: {
         files: { src: 'src/ngCookies/**/*.js' },
       },
-      ngLocale: {
```
Comment (Narretz) [inline Gruntfile.js]: Why is this removed?

Comment (lgalfaso) [inline Gruntfile.js]: The function `pluralCat` that comes from google closure does not pass the jshint.

**Comment by petebacondarwin** [comment]
Closing in favour of #12362
